generator/04-syndesis-atlasmap.yml.mustache

```diff
@@ -70,35 +70,10 @@
           app: syndesis
           component: syndesis-atlasmap
           deploymentconfig: syndesis-atlasmap
-        annotations:
```

Comment:
This has been removed since OCP 3.6 fully supports init-containers via the API directly. Is this correct?